CHANGELOG.md

```diff
@@ -23,6 +23,23 @@
 9. [Continuous integration](https://travis-ci.org/github/idontgetoutmuch/random).
 10. Fully documented - for more details see the [haddock](https://htmlpreview.github.io/?https://github.com/idontgetoutmuch/random/blob/release-notes/docs/System-Random.html).
 
+### Issues Addressed
+
+ Issue Number | Description | Comment
+--------------|-------------|--------
+ [25](https://github.com/haskell/random/issues/25) | The seeds generated by split are not independent |
+ [26](https://github.com/haskell/random/issues/26) | Add Random instances for tuples |
+ [44](https://github.com/haskell/random/issues/44) | Add Random instance for Natural |
+ [51](https://github.com/haskell/random/issues/51) | Very low throughput |
+ [53](https://github.com/haskell/random/issues/53) | incorrect distribution of randomR for floating-point numbers | Fixed
```

Comment:
```suggestion
 [53](https://github.com/haskell/random/issues/53) | incorrect distribution of randomR for floating-point numbers | Documented behaviour of floating point numbers
```

We haven't actually _done_ this yet :) I'll create a PR.

Reply:
Ooh yes we meant to do it - thanks

Reply:
I've started writing a little Haddock section for this. I'll create a PR once I have a draft.